### PR TITLE
Fix: with 1-bus only result, reset result uuid and delete logs

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/dto/ComputationType.java [hunk 1]
```diff
@@ -1,16 +1,10 @@
 package org.gridsuite.study.server.dto;
 
 public enum ComputationType {
-    LOAD_FLOW("LoadFlow"),
-    SECURITY_ANALYSIS("SecurityAnalysis"),
-    SENSITIVITY_ANALYSIS("SensitivityAnalysis"),
-    SHORT_CIRCUIT("ShortCircuitAnalysis"),
-    VOLTAGE_INITIALIZATION(""),
-    DYNAMIC_SIMULATION("");
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/server/dto/ComputationType.java:9]: Maybe it's useful for deserialization in SupervisionController ? Not sure though

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 2]
```diff
@@ -138,7 +138,6 @@ public enum ReportNameMatchingType {
         EXACT_MATCHING, ENDS_WITH
     }
 
-    // TODO remove ComputationType (and use ReportType instead) after short-circuit logs have been invalidaded with admin-tool
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:141]: Shouldn't we keep it as ComputationType is not completly removed ?

File: src/main/java/org/gridsuite/study/server/service/SupervisionService.java [hunk 3]
```diff
@@ -175,24 +178,35 @@ private Integer deleteSensitivityAnalysisResults() {
...
-        nodes.stream().forEach(node -> {
-            node.setShortCircuitAnalysisResultUuid(null);
-            node.setOneBusShortCircuitAnalysisResultUuid(null);
-        });
-        Map<UUID, String> subreportToDelete = formatSubreportMap(ComputationType.SHORT_CIRCUIT.subReporterKey, nodes);
-        reportService.deleteTreeReports(subreportToDelete);
+        // Reset result uuid and remove logs, for all-buses computations, then for 1-bus ones
+        List<NetworkModificationNodeInfoEntity> allBusesNodes = networkModificationNodeInfoRepository.findAllByShortCircuitAnalysisResultUuidNotNull();
+        if (!allBusesNodes.isEmpty()) {
+            allBusesNodes.stream().forEach(node -> node.setShortCircuitAnalysisResultUuid(null));
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/server/service/SupervisionService.java:184]: you can remove .stream()

File: src/main/java/org/gridsuite/study/server/service/SupervisionService.java [hunk 4]
```diff
@@ -175,24 +178,35 @@ private Integer deleteSensitivityAnalysisResults() {
...
+            reportService.deleteTreeReports(formatSubreportMap(StudyService.ReportType.ALL_BUSES_SHORTCIRCUIT_ANALYSIS.toString(), allBusesNodes));
+        }
+        List<NetworkModificationNodeInfoEntity> oneBusNodes = networkModificationNodeInfoRepository.findAllByOneBusShortCircuitAnalysisResultUuidNotNull();
+        if (!oneBusNodes.isEmpty()) {
+            oneBusNodes.stream().forEach(node -> node.setOneBusShortCircuitAnalysisResultUuid(null));
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/server/service/SupervisionService.java:189]: Same

File: src/main/java/org/gridsuite/study/server/service/SupervisionService.java [hunk 5]
```diff
@@ -175,24 +178,35 @@ private Integer deleteSensitivityAnalysisResults() {
...
     }
 
     private Integer deleteVoltageInitResults() {
         AtomicReference<Long> startTime = new AtomicReference<>();
         startTime.set(System.nanoTime());
         List<NetworkModificationNodeInfoEntity> nodes = networkModificationNodeInfoRepository.findAllByVoltageInitResultUuidNotNull();
-        nodes.stream().forEach(node -> node.setVoltageInitResultUuid(null));
-        //TODO Add logs deletion once they are added
+        if (!nodes.isEmpty()) {
+            nodes.stream().forEach(node -> node.setVoltageInitResultUuid(null));
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/server/service/SupervisionService.java:207]: Same

File: src/main/java/org/gridsuite/study/server/dto/ComputationType.java (hunk 1 above)
Comment (dbraquart) [reply src/main/java/org/gridsuite/study/server/dto/ComputationType.java:9]: I think it's safe to remove the "subReporterKey" here

File: src/main/java/org/gridsuite/study/server/service/StudyService.java (hunk 2 above)
Comment (dbraquart) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java:141]: Sadly we have to keep the two enums, so the TODO is useless.

**Comment by dbraquart** [comment]
I also made a change about naming (discussed with @SlimaneAmar)

**Review by TheMaskedTurtle — APPROVED** [review]

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**